ARTCC/Danger Areas/EG D323.txt

```diff
@@ -1,42 +1,86 @@
 ;EGD323A
-EGD323A    N055.17.10.000 E000.14.28.000 N055.07.20.000 E000.55.08.000
-EGD323A    N055.07.20.000 E000.55.08.000 N054.27.08.000 W000.15.45.000
-EGD323A    N054.27.08.000 W000.15.45.000 N054.33.16.000 W000.21.06.000
-EGD323A    N054.33.16.000 W000.21.06.000 N054.44.13.000 W000.45.45.000
-EGD323A    N054.44.13.000 W000.45.45.000 N054.57.44.000 W000.54.57.000
-EGD323A    N054.57.44.000 W000.54.57.000 N055.17.10.000 E000.14.28.000
+EGD323A    N055.17.10.000 E000.14.28.000 N055.01.43.000 E001.17.53.000
+EGD323A    N055.01.43.000 E001.17.53.000 N054.31.02.000 W000.31.32.000
+EGD323A    N054.31.02.000 W000.31.32.000 N054.48.41.000 W000.54.56.000
+EGD323A    N054.48.41.000 W000.54.56.000 N054.57.44.000 W000.54.57.000
+EGD323A    N054.57.44.000 W000.54.57.000 N054.57.44.000 W000.54.57.000
+EGD323A    N055.17.10.000 E000.14.28.000 N055.17.10.000 E000.14.28.000
```

Comment:
This line is not needed - it's simply drawing a line between the same two points